Fixes #372

idogi opened the pull request.

danielhers commented:
Thanks. It's great that you say which issue you fixed, but please also give a brief description of what you did. 

Did you test that all the tour stages still work properly? I remember some of them not showing when I tried fixing this.

idogi commented:
It seems that the bug stemmed from an unnecessary tour.restart() call.
Removed it and changed the following start() to a restart().

Al stages were tested to appear in order.
 
Tour still has some issues. see #393.

omerxx commented on static/js/tour.js:
```diff
@@ -95,12 +95,12 @@ function tourClick(){
     checkStepsAdd = false;
     checkStepsAdd2 = false;
-    tour.restart();
+    //tour.restart();
```
Is the comment necessary?

danielhers replied:
Probably not.

danielhers commented:
@idogi, could you make that description the commit message by doing commit --amend and push -f?

omerxx commented:
We can also change it inline here directly once it's merged..

omerxx commented:
I deleted the comment manually
